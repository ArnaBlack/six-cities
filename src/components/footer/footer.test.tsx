import * as React from 'react';
import {
  configure,
  shallow,
} from 'enzyme';
import * as Adapter from 'enzyme-adapter-react-16';
import Footer from './footer';

configure({adapter: new Adapter()});

it(`Footer renders correctly`, () => {
  const tree = shallow(<Footer />);

  expect(tree).toMatchSnapshot();
});
