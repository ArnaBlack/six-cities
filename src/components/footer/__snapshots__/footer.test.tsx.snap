// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders correctly 1`] = `
<footer
  className="footer container"
>
  <NavLink
    className="footer__logo-link"
    to="main.html"
  >
    <img
      alt="6 cities logo"
      className="footer__logo"
      height="33"
      src="img/logo.svg"
      width="64"
    />
  </NavLink>
</footer>
`;
